app/javascript/components/widgets/weather/weather-effect.jsx

```diff
@@ -319,17 +320,36 @@ const MovingCloud = styled.div`
   ${CloudImage};
 `;
 
-const Cloud = ({ duration, variance, count }) => (
-  <HorizontalWeatherWrapper
-    {...{ Type: MovingCloud, duration, variance, count }}
-  />
-);
+const Cloud = ({ duration, variance, count, windSpeed }) => {
+  // eslint-disable-next-line no-shadow
+  if (windSpeed <= 10) {
+    return (
+      <HorizontalWeatherWrapper
+        {...{ Type: MovingCloud, duration: duration + 15, variance, count }}
+      />
+    );
+  }
+  if (windSpeed >= 15) {
+    return (
+      <HorizontalWeatherWrapper
+        {...{ Type: MovingCloud, duration: duration - 5, variance, count }}
+      />
+    );
+  }
+  return (
+    <HorizontalWeatherWrapper
+      {...{ Type: MovingCloud, duration, variance, count }}
+    />
+  );
```

Comment:
Nit: this is a little odd that we have a default duration that is for winds > 10 and < 15. Defaulting to the slowest wind duration and decreasing the duration for faster wind speeds may be a little more intuitive.